docs/developer-docs/latest/development/backend-customization/policies.md

```diff
@@ -0,0 +1,168 @@
+---
+title: Backend customization policies - Strapi Developer Documentation
+description: The backend of Strapi can be customized according to your needs, so you can create your own backend behavior.
+---
+<!-- TODO: update SEO -->
+
+# Policies
+
+Policies are functions which have the ability to execute specific logic on each request before it reaches the controller's action. They are mostly used for securing business logic easily.
```

Comment:
To check with @alexandrebodin but it should change. Policy will only be functions that return `true` or `false` (in order to access the controller or not). In order to do specific logic before the controller, middlewares should be used instead.